changed.d

```diff
@@ -384,17 +386,20 @@ Please supply a bugzilla version
         writeln("Skipped querying Bugzilla for changes. Please define a revision range e.g ./changed v2.072.2..upstream/stable");
     }
 
+    if (useNightlyTemplate)
+    {
+        stderr.writeln("--nightly is deprecated. Use --release-type=nightly instead.");
+        releaseType = "nightly";
+    }
+
     auto f = File(outputFile, "w");
     auto w = f.lockingTextWriter();
     w.put("Ddoc\n\n");
     w.formattedWrite("$(CHANGELOG_NAV_LAST %s)\n\n", previousVersion);
 
     {
-        // NITGHLY_VERSION is a special ddoc macro with e.g. different download links
-        if (useNightlyTemplate)
-            w.formattedWrite("$(NIGHTLY_VERSION %s,\n,\n,", nextVersionDate);
-        else
-            w.formattedWrite("$(VERSION %s, =================================================,\n\n", nextVersionDate);
+        import std.uni : asUpperCase;
+        w.formattedWrite("$(%sVERSION %s, =================================================,\n\n", text(releaseType.asUpperCase, "_"), nextVersionDate);
```

Comment:
Why append the underscore to the `releaseType` separately, instead of putting the underscore in the format string? Did you mean to make the `releaseType` including underscore optional?